README.md

```diff
@@ -35,6 +35,7 @@ readme_index:
   enabled:          true
   remove_originals: false
   with_frontmatter: false
+  regex:            null
```

Comment:
I wasn't sure if to use `nil` or `null` here but I got no syntax highlighting for `nil` and yes syntax highlighting for `null` in VS Code so I chose the latter.